Classes/common/HTTP/CDTURLSessionTask.m

```diff
@@ -18,6 +18,12 @@
 #import "CDTHTTPInterceptor.h"
 #import "CDTLogging.h"
 
+@interface CDTHTTPInterceptorContext ()
+
+@property (nullable, nonatomic, strong) NSURLResponse *response;
```

Comment:
All properties on `CDTHTTPInterceptorContext` are readwrite as of commit d18dea9